### ISSUE TITLE
[💡 Feature]: Provide examples and docs for testing Angular components in component tests

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#9912]
### Is your feature request related to a problem?

I think there is not much needed to get support for Angular. No specific preset required. However we should add examples to the docs to let users know how to integrate Angular.

### Describe the solution you'd like.

- add an Angular example in https://github.com/webdriverio/webdriverio/tree/main/examples/wdio/browser-runner
- add Angular frame setup docs https://webdriver.io/docs/component-testing/lit
- add Angular logo to the front page section on `Easy setup for web component testing with:`
- add Angular example to https://github.com/webdriverio/component-testing-examples

### Describe alternatives you've considered.

n/a

### Additional context

n/a

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sharath2106** [comment]
@christian-bromann I would like to work on this if this is not picked up.

**Comment by christian-bromann** [comment]
Awesome, let me know if you have any questions.

**Comment by hughleo** [comment]
Really interested in this...have you been able to make any progress @sharath2106 ?

**Comment by sharath2106** [comment]
@hughleo Yes. I have made some progress. I'll raise the PR.

**Comment by christian-bromann** [comment]
Any updates @sharath2106 ?

**Comment by udarrr** [comment]
Maybe some kind of off top but anyway. Can we test angular with wdio? Because I am little bit confused. It seems wdio don't support angular component but here would be provided some info regarding that.

**Comment by christian-bromann** [comment]
> Can we test angular components with wdio?

Yes they can. I am not sure if we need a preset with some additional Vite configurations to make it simpler for everyone but in general it is possible. For some frameworks like Svelte or Vue we needed extra plugins to be installed which is why there are presets. They configure Vite to be able to load these components without needing much setup from the user side.

**Comment by udarrr** [comment]
If it were so easy It would be great and an example could be really helpful here. As i know only one framework was able implement browser related to components tests in angular currently, and others already are working on that. I'd like to help with that but unfortunately I'm not so experienced with angular.

**Comment by mumenthalers** [comment]
@sharath2106 any progress?
